docs/resie_energy_system_components.md

```diff
@@ -594,13 +594,93 @@ Modelled as [fuel boiler](resie_energy_system_components.md#fuel-boiler-fb) with
+
+$${LMTD} = \frac{\Delta T_A - \Delta T_B}{\ln{\Delta T_A} - \ln{\Delta T_B}}$$
+$$\Delta T_A = T_{source,in} - T_{sink,out}$$
+$$\Delta T_B = T_{source,out} - T_{sink,in}$$
+
+for a counter-current heat exchanger. Similar calculations can be made for cocurrent and cross-current heat exchangers. Setting \(\Delta T_B = \alpha \cdot \Delta T_A\) allows us to disregard the unknown variables by introducing a new one in the form of \(\alpha\), which is a measure for the temperature spread across the length of the exchange area. As heat exchangers are designed for a specific range in temperatures and mass flow, \(\alpha\) is expected to change in respect to the temperatures, becoming smaller the more \(T_{source,in}\) approaches the extreme sides of the range.
+
+This leads to the following formulation:
+
+$$T_{source,in,avg} = \frac{1}{2} \cdot (T_{source,in,max} + T_{source,in,min})$$
+$$\alpha(T_{source,in}) = 1 - \frac{abs(T_{source,in,avg} - T_{source,in})}{(T_{source,in,max} - T_{source,in,min})}$$
+$$\alpha_c = min(\alpha_{max},\ \alpha(T_{source,in}))$$
+$$T_{sink,out} = T_{source,in} - L_{min} \cdot \frac{\ln{\alpha_c^{-1}}}{1-\alpha_c}$$
+
+with \([T_{source,in,min},\ T_{source,in,max}]\) being the design temperature range and \(L_{min} \cdot \alpha_{max}^{-1}\) being the minimal reduction in temperature.
+
+**Note:** This is not a fully sound formulation, but has been deemed sufficiently accurate to values from experience.
+
+## Heat sources
+Some sources of heat, such as ground sources, require detailed models and are described in their own sections. A generic implementation for heat sources is provided, which can model any type of heat source which only provides heat, but does not take any in. Ground sources typically must be regenerated outside of the heating period and thus act as a seasonal storage of heat.
+
+**Typical sources of heat**
+
+heat medium | name | implementation
+--- | --- | ---
+ground | probes | [Geothermal Probes](resie_energy_system_components.md#geothermal-probes)
+ground | horizontal collector | [Geothermal Heat Collector](resie_energy_system_components.md#geothermal-heat-collector)
+ground | basket collector | not implemented
+ground | trensh collector | not implemented
+ground | spiral collector | not implemented
+water | groundwater well | [Generic Heat Source](resie_energy_system_components.md#generic-heat-source)
+water | surface waters | [Generic Heat Source](resie_energy_system_components.md#generic-heat-source)
+water | waste heat from industrial processes | [Generic Heat Source](resie_energy_system_components.md#generic-heat-source)
+water | waste water | [Generic Heat Source](resie_energy_system_components.md#generic-heat-source)
+water | solar thermal collector | [Generic Heat Source](resie_energy_system_components.md#generic-heat-source)
```

Comment:
Should we change this to a (currrently emtpy) chapter that will be written soon to keep in mind the link?

Reply:
It links to the description of the model, which comes with this PR.